src/_pytest/config/__init__.py

```diff
@@ -521,22 +521,29 @@ def import_plugin(self, modname):
         modname = str(modname)
         if self.is_blocked(modname) or self.get_plugin(modname) is not None:
             return
-        if modname in builtin_plugins:
-            importspec = "_pytest." + modname
-        else:
-            importspec = modname
+
+        importspec = "_pytest." + modname if modname in builtin_plugins else modname
         self.rewrite_hook.mark_rewrite(importspec)
+
+        attempt_entry_point = modname not in builtin_plugins
+        if attempt_entry_point:
+            infos = len(self.list_plugin_distinfo())
+            new_infos = self.load_setuptools_entrypoints("pytest11", name=modname)
+            loaded = new_infos > infos
```

Comment:
😷

Reply:
this allows 2 extra kex features a s followup

a) putting pytests own plugins into entrypoints (using a own entrypoint limited to pytest core)
b) inventing a naming mechanism for plugins that do not autoload, but should have a short name to autoload

it may be necessary to go a step further on pluggy and externalizing the plugin loader concept for a better behavioral envelope